dags/experiments_live.py

```diff
@@ -89,10 +87,12 @@
             date_partition_parameter=None,
             depends_on_past=True,
             dag=dag,
+            is_delete_operator_pod=True,
```

Comment:
This should get passed into the `GKEPodOperator` as parameter: https://github.com/mozilla/telemetry-airflow/blob/17a87679dce6bd4ab84ca840378daf64c0debc0e/dags/utils/gcp.py#L238